Migration script/openIMIS migration v1.4.2 - 1.5.0.sql

```diff
@@ -1291,3 +1291,8 @@ BEGIN
 	RETURN 0
 END
 GO
+
+
+-- OP-238: Discrepancy in reporting of IMIS Policies
+INSERT [dbo].[tblRoleRight] ([RoleID], [RightID], [ValidityFrom], [ValidityTo], [AuditUserId], [LegacyID]) 
+VALUES ((select RoleID from tblRole where RoleUUID='8B7F95F6-065F-47D7-8454-BF7B82FBB301'), 131201, CURRENT_TIMESTAMP, NULL, NULL, NULL);
```

Comment:
Looking on the DB creation scripts, there is no role with that RoleUUID.  I suggest to either use the RoleID 1 as in the first commit or search per IsSystem=1.

Reply:
Please solve the conflicts.

Reply:
@dragos-dobre There are two record with IsSystem=1. One named "Enrolment Officer" and second "Enrol**e**ment Officer". Also EO don't always have RoleID == 1 (in BEPHA it's 13 and 33), so I've user select statement that will update both records.